dev/ory/oathkeeper.yml

```diff
@@ -9,6 +9,15 @@ tracing:
       local_agent_address: otel-agent:6831
 
 authenticators:
+  jwt:
+    enabled: true
+    config:
+      trusted_issuers:
+        - https://firebaseappcheck.googleapis.com/1076318732193
```

Comment:
@nicolasburtey need to swap out the number `1076318732193` for the value from galoy's `firebase > project settings > Project Number`